user-guide/Standard_Apps/Monitoring_Solutions/Nevion Video IPath/NevionVideoIPath_Overview.md

```diff
@@ -0,0 +1,28 @@
+---
+uid: Overview_NevionVideoIPath_App
+---
+
+# Nevion Video IPath app
+
+The Nevion Video IPath app allows you to control Sony's (Nevion) VideoIPath SDN controller, which provides connection management, service assurance, and network inventory capabilities for service provides and broadcasters.
+
+![Nevion Video IPath app](~/user-guide/images/Nevion_OverviewPage.png)
+
+The Nevion Video IPath app retrieves all data from the Nevion VideoIPath system using the [Nevion Video iPath connector](https://catalog.dataminer.services/details/7b986cbc-a29e-44e5-b3b7-003e639acfac), using Nevion’s modern services API calls. It then allows you to schedule, execute, and view connections between endpoints. This will not only establish a routing path but also configures a service that could be set up as point-to-point or point-to-multipoint.
+
+The [*Nevion Video IPath* package](https://catalog.dataminer.services/details/ec97a934-679e-42c2-ac4b-f33a3d9f8d2c), available for download from the DataMiner Catalog, contains the following items:
+
+- The Nevion Video IPath low-code app
+
+- Nevion Video IPath connector version 1.3.0.7
```

Comment:
Does the version have to be mentioned here? This is information that will probably need to be updated a lot, which is likely to get forgotten at some point. If it's not necessary, I'd therefore suggest to remove the version info here.

Reply:
Is it ok then if I say "Nevion Video IPath connector version 1.3.0.7 or higher"? I believe it's important to specify a minimum version to ensure that all features are included.